src/components/shared/Entity/Details/useDetailsEntityTaskTypes.ts

```diff
@@ -0,0 +1,22 @@
+import { useEntityType } from 'context/EntityContext';
+import { useMemo } from 'react';
+import { ShardEntityTypes } from 'stores/ShardDetail/types';
+import useIsCollectionDerivation from './useIsCollectionDerivation';
+
+function useDetailsEntityTaskTypes() {
```

Comment:
Should this hook be named `useEntityTaskTypes` to lessen the likelihood renaming in the future if it is used outside the context of the _Details_ page?

Reply:
I do not think so. I added the `Details` into it to make sure it would not end up outside of details as "easily". If we do move this out I kinda want it to be a "bigger change" that includes renaming the hook.